allensdk/brain_observatory/nwb/ndx-aibs-behavior-ophys.extension.yaml

```diff
@@ -3,112 +3,84 @@ groups:
   neurodata_type_inc: LabMetaData
   doc: Metadata for behavior or behavior + ophys task parameters
   attributes:
+  - name: task
```

Comment:
Would there be a way to consolidate the experiment descriptions into this file (or something like it)? It is inconsistent that some metadata is hard-coded into a python function, and other are in these accompanying files.

Or alternatively -- maybe a config-type file that contains that information (and can import it for tests + the truncate/match function) rather than putting this data in the function itself. I think it should be obvious where it comes from and not necessarily involve a code change to update.

Reply:
Yes, I can move the experiment descriptions to a separate config-style file. The pyNWB behavior-ophys extension would probably not be the right place to put that information though :/

Another thought I had was, would it make sense to store experiment descriptions in LIMS and have it pass that info down? That would probably be the 'kosher' way to do it, but seemed to me like it would involve a protracted process.